core/src/main/java/com/novoda/noplayer/internal/exoplayer/ExoPlayerFacade.java

```diff
@@ -29,22 +32,27 @@
     private final MediaSourceFactory mediaSourceFactory;
     private final ExoPlayerAudioTrackSelector audioTrackSelector;
     private final ExoPlayerSubtitleTrackSelector subtitleTrackSelector;
+    private ExoPlayerVideoTrackSelector exoPlayerVideoTrackSelector;
```

Comment:
if isn't final, can you move it after the final ones?

Reply:
I think my IDE settings are messed up, this should have happened automatically when I extracted the field 😢

Reply:
b8c8f48